roadmap.md

```diff
@@ -0,0 +1,38 @@
+Roadmap Zombienet v2
+
+## Infra
+- Chaos testing, add examples and explore possibilities in `native` and `podman` provider
+- Add `docker` provider
+- Add `nomad` provider
+- Create [helm chart](https://helm.sh/docs/topics/charts/) to allow other use zombienet in k8s
+- Auth system to not use k8s users
+- Create GitHub Action and publish in marketplace (wip)
+
+## Internal teams
+- Add more teams (wip)
+
+## UI
+- Create UI to create `.feature` (`.zndls`) and `network` files.
+- Create script to translate from polkadot launch config to zombienet network file. [Open PR](https://github.com/paritytech/zombienet/tree/nik-convert-polkadot-launch-config-to-zombienet) (once merged we can remove from roadmap)
```

Comment:
This is done and PR is merged. Should we remove it from the roadmap already @pepoviola ?
```suggestion
```

Reply:
Yes, thanks @wirednkod